OPHD/States/MapViewState.cpp

```diff
@@ -411,27 +411,34 @@ void MapViewState::onKeyDown(NAS2D::EventHandler::KeyCode key, NAS2D::EventHandl
 		mReportsUiSignal();
 		return;
 	}
-
+	int repeat = 1;
+	if ((uint16_t)mod & (uint16_t)NAS2D::EventHandler::KeyModifier::Shift)
```

Comment:
Bit of a style issue here, we avoid C-style casts, and instead use C++ style casts. For example `static_cast<uint16_t>(mod)`.

However, in this particular case, I suspect an explicit cast may not be necessary. There are overloaded logical operators for this type, so it should be possible to do the logical `&` without needing to cast. See `EventHandler.h` and `EventHandler.cpp`.